src/users/users.service.ts

```diff
@@ -0,0 +1,51 @@
+import { Injectable } from '@nestjs/common';
+import { CreateUserDto } from './dto/createUser.dto';
+//import { UserRepository } from './user.repository';
+import { InjectRepository } from '@nestjs/typeorm';
+import { User } from './user.entity';
+import { Repository } from 'typeorm';
+import { UpdateUserDto } from './dto/updateUser.dto';
+
+@Injectable()
+export class UsersService {
+  constructor(
+    @InjectRepository(User)
+    private usersRepository: Repository<User>,
+  ) {}
+  // constructor(
+  //   @InjectRepository(UserRepository)
+  //   private userRepository: UserRepository,
+  // ) {}
+  async createUser(createUserDto: CreateUserDto) {
+    const { userName, password, name, email } = createUserDto;
+    const user = new User();
+    user.userName = userName;
+    user.password = password;
+    user.name = name;
+    user.email = email;
```

Comment:
可以用 create 產生 https://typeorm.io/repository-api
```suggestion
    const userDraft = await this.usersRepository.create(createUserDto);
     await this.usersRepository.insert(userDraft);

```